deploy-commands.js

```diff
@@ -1,5 +1,8 @@
 const { REST, Routes } = require('discord.js');
-const { clientId, guildId, token } = require('./config.json');
+//const { clientId, guildId, token } = require('./config.json');
```

Comment:
will have to change a lot of stuff if we go from config.json -> .env, especially how we build. pros /cons of this?

Reply:
Pros: github action automate this process once we have CI configured. Cons: have to change github secrets if we change anything in config.json